language/resource-api/README.md

```diff
@@ -143,6 +142,8 @@ software { php:
 }
 ```
 
+The provider should return a string containing all namevar values in a string matching one of the title patterns. This allows the runtime environment to present resources in a way familiar to the user.
```

Comment:
```suggestion
The provider returns a string containing all namevar values in a string matching one of the title patterns.
```

Reply:
I don't think the last line adds anything, and avoid saying "the user", as they are the ones reading it.

Reply:
Actually, now that this sentence was called out and I read it again, it does not really make sense - "returning a string containing all namevar values in a string" ... what does that actually mean. And the original said "should return", as in a requirement on the author of a provider; unclear if this is something an implementor of a resource type can do anything about...

Can meaning be made a more clear?

Reply:
> I don't think the last line adds anything, and avoid saying "the user", as they are the ones reading it.

For clarity, this documentation is directed at a module author, they must ensure that the provider that they are writing returns a `namevar` string that matches a `title-pattern` in a way that  "the user" (the eventual end user of the new module) understands.

Reply:
I've rewritten this section extensively